CONTRIBUTING.md

```diff
@@ -12,51 +12,64 @@ contributing enhancements to the Prometheus charm.
   implementation, you can reach us at
   [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)
   or [Discourse](https://discourse.charmhub.io/).
-  The primary author of this charm is available on the Mattermost channel as
-  `@balbir-thomas`.
 - Familiarising yourself with the
   [Charmed Operator Framework](https://juju.is/docs/sdk)
   library will help you a lot when working on new features or bug fixes.
 - All enhancements require review before being merged. Code review
   typically examines
   + code quality
   + test coverage
-  + user experience for Juju administrators
-  this charm.
+  + user experience for Juju administrators.
 - Please help us out in ensuring easy to review branches by rebasing
-  your pull request branch onto the `main` branch. This also avoids
-  merge commits and creates a linear Git commit history.
+  your pull request branch onto the `main` branch. This also avoids merge commits and creates a linear Git commit history.
 
-## Developing
 
-Create and activate a virtualenv with the development requirements:
+## Setup
 
-```bash
-$ virtualenv -p python3 venv
-$ source venv/bin/activate
-```
+A typical development setup for charms created with the [Charmed Operator Framework](https://juju.is/docs/sdk) contains:
+
+
+- [Charmcraft](https://github.com/canonical/charmcraft) - Developer tooling for creating, building and publishing Charmed Operators
+- [Juju](https://juju.is/) - a Charmed Operator Lifecycle Manager (OLM), used for deploying and managing operators
+- [Multipass](https://multipass.run/) - a lightweight Ubuntu virtual machine manager (optional)
+- [MicroK8s](https://microk8s.io/) - a low-ops Kubernetes distribution we’ll use for testing our Charmed Operator (optional if developing a Charmed Operator for Kubernetes)
+
+Please [follow this guide](https://juju.is/docs/sdk/dev-setup) which will walk through the installation of these tools to get you started with charm development.
```

Comment:
We should probably offload as much as sensible to the dev-setup doc.
How about:

```suggestion
This prometheus operator is developed with the [Charmed Operator Framework](https://juju.is/docs/sdk).

Please follow the [dev-env setup guide](https://juju.is/docs/sdk/dev-setup) to get started.
```